src/store/indexeddb-local-backend.ts

```diff
@@ -127,6 +127,8 @@ export class LocalIndexedDBStoreBackend implements IIndexedDBBackend {
     private db: IDBDatabase = null;
     private disconnected = true;
     private _isNewlyCreated = false;
+    private _isPersisting = false;
+    private _pendingUserPresenceData: UserTuple[] = [];
```

Comment:
I think we don't prefix private members with `_` anymore; the only reason it is used for `_isNewlyCreated` is that there is a getter with the name `isNewlyCreated`